src/core/lender/tranche/operator.sol

```diff
@@ -36,17 +36,30 @@ contract Operator is DSNote,DSMath {
     modifier auth { require(wards[msg.sender] == 1); _; }
 
     // -- Users --
-    function addUser(address usr) public auth note { wards[usr] = 2; }
+    uint constant UserAccess = 2;
+    function addUser(address usr) public auth note { wards[usr] = UserAccess; }
     function denyUser(address usr) public auth note { wards[usr] = 0; }
-    modifier auth_external { require(wards[msg.sender] != 0); _; }
+
+    // auth_external controls supply and redeem access
+    // either openAccess or only for admins and users
+    modifier auth_external { require(openAccess || wards[msg.sender] != 0); _; }
```

Comment:
If you call it investors, 
```suggestion
    modifier auth_investor { require(investors[msg.sender] != 0); _; }
```
or if you stick to the term user:

```suggestion
    modifier auth_user { require(users[msg.sender] != 0); _; }
```

Reply:
used investor